mne/evoked.py

```diff
@@ -883,42 +884,45 @@ def combine_evoked(all_evoked, weights):
     -----
     .. versionadded:: 0.9.0
     """
+    naves = np.array([evk.nave for evk in all_evoked], float)
     if isinstance(weights, str):
         _check_option('weights', weights, ['nave', 'equal'])
         if weights == 'nave':
-            weights = np.array([e.nave for e in all_evoked], float)
-            weights /= weights.sum()
-        else:  # == 'equal'
-            weights = [1. / len(all_evoked)] * len(all_evoked)
-    weights = np.array(weights, float)
+            weights = naves / naves.sum()
+        else:
+            weights = np.ones_like(naves)
+    else:
+        weights = np.array(weights, float)
+
     if weights.ndim != 1 or weights.size != len(all_evoked):
         raise ValueError('weights must be the same size as all_evoked')
 
+    # cf. https://en.wikipedia.org/wiki/Weighted_arithmetic_mean, section on
+    # how variances change when summing Gaussian random variables. The variance
+    # of a weighted sample mean is:
+    #
+    #    σ² = w₁² σ₁² + w₂² σ₂² + ... + wₙ² σₙ²
+    #
+    # We estimate the variance of each evoked instance as 1 / nave to get:
+    #
+    #    σ² = w₁² / nave₁ + w₂² / nave₂ + ... + wₙ² / naveₙ
+    #
+    # And our resulting nave is the reciprocal of this:
+    new_nave = 1. / np.sum(weights ** 2 / naves)
+    # This general formula is equivalent to formulae in Matti's manual
+    # (pp 128-129), where:
```

Comment:
sorry late to the party, but Matti's manual is now merged with the online documentation. There is in fact no pdf that you can find from the MNE website. Thus I think it's better to just link to the online documentation.

Reply:
>  There is in fact no pdf that you can find from the MNE website.

When I log in to [the download page for MNE-C](http://www.nmr.mgh.harvard.edu/martinos/userInfo/data/MNE_register/login.php) I see `MNE-manual-2.7.3.pdf`

Reply:
Oh I see. It wasn't obvious to me :)